tox.ini

```diff
@@ -105,11 +101,7 @@ deps =
     typing_extensions~=4.2
     jsonpatch~=1.33
     -e .
-    # TODO: If the code in testing changes, it isn't reinstalled.
-    # -e doesn't install it with the right name. We should probably
-    # build a wheel and then use that across the environments,
-    # similar to what was done in the ops-scenario repository.
-    ./testing
+    -e ./testing/.
```

Comment:
`./testing/.` looks funky, but I wonder if this notation is required.

Reply:
> `./testing/.` looks funky, but I wonder if this notation is required.

It can't be `testing` (because that would look for a package on PyPI called "testing"). `./testing` and `testing/.` both work for me, at least with `uv pip`. I don't have a strong preference here - would you prefer either of those?

Reply:
Actually, now that `-e` is there, `testing` would work because the `-e` forces it to be a local reference since you can't have an editable PyPI install.

So it could be `-e testing` if you prefer that? There's a small risk that someone drops the `-e` or similar and causes problems in the future, but I think `testing` is a restricted PyPI name anyway, so people can't actually register something bad under it.